.github/workflows/R-CMD-check.yaml

```diff
@@ -27,14 +27,6 @@ jobs:
       matrix:
         config:
           - {os: macOS-latest,   r: 'release'}
-          - {os: windows-latest, r: 'release'}
```

Comment:
@shamindras are we going to add this line later?

Reply:
@ricfog - yes! We will add this back in after we have added in more features.